fireTorpedo(ALL) implemetation missing

I tried to test the project with Travis CI, but i get:
[ERROR]   GT4500Test.fireTorpedo_All_Success:37 expected: <true> but was: <false>,
because the fireTorpedo(FiringMode firingMode) function doesn't have an implemetation for the firingMode value 'ALL'.

--- a/README.md
+++ b/README.md
@@ -36,3 +36,4 @@ The project represents an alpha version of a spaceship.
 - Lasers are not yet implemented, but the code for torpedo stores are ready (`TorpedoStore`).
 - For the GT4500 ship the rules for firing torpedoes can be found in the Javadoc comment of method `fireTorpedos`. They are already partially implemented.
 - There are currently two tests (`GT4500Test`), but be aware that they are not proper unit tests, as they do not isolate the dependencies of the tested class.
+.....

--- a/src/main/java/hu/bme/mit/spaceship/GT4500.java
+++ b/src/main/java/hu/bme/mit/spaceship/GT4500.java
@@ -78,8 +78,10 @@ public class GT4500 implements SpaceShip {
 
       case ALL:
         // try to fire both of the torpedo stores
-        //TODO implement feature
 
+	   if (!secondaryTorpedoStore.isEmpty() && !primaryTorpedoStore.isEmpty()) {
+            firingSuccess = secondaryTorpedoStore.fire(1) && primaryTorpedoStore.fire(1);
+        }
         break;
     }
 

